sky/clouds/__init__.py

```diff
@@ -1,18 +1,20 @@
 """Clouds in Sky."""
-from sky.clouds.cloud import Cloud
-from sky.clouds.cloud_registry import CLOUD_REGISTRY
-from sky.clouds.cloud import CloudImplementationFeatures
-from sky.clouds.cloud import Region
-from sky.clouds.cloud import Zone
+# TODO: isort:skip is ugly. We should have a better way to do this to avoid
+# circular imports.
+from sky.clouds.cloud_registry import CLOUD_REGISTRY  # isort:skip
+from sky.clouds.cloud import Cloud  # isort:skip
+from sky.clouds.cloud import CloudImplementationFeatures  # isort:skip
+from sky.clouds.cloud import Region  # isort:skip
+from sky.clouds.cloud import Zone  # isort:skip
 from sky.clouds.aws import AWS
```

Comment:
```suggestion
from sky.clouds.cloud import Cloud
from sky.clouds.cloud import CloudImplementationFeatures
from sky.clouds.cloud import Region
from sky.clouds.cloud import Zone
from sky.clouds.cloud_registry import CLOUD_REGISTRY

# NOTE: import the above first to avoid circular imports.
# isort: split
from sky.clouds.aws import AWS
```
Nice! Can consider adding an `# isort: split` below L8?